0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -213,6 +214,13 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
+#### rejectIncomingTransfer
+<code>ledgerPlugin.rejectIncomingTransfer( **transferId**:String ) ⇒ Promise.&lt;null></code>
```

Comment:
Another parameter.

Though this may require some more systematic discussion. We have `fulfill` (success), `rejectIncomingTransfer` (failure) and `replyToTransfer` (neutral). Right now, we can send messages only with the latter.

* What use cases are there for `replyToTransfer`?
* Is there a use case for returning a message with success (`fulfill`)?